### ISSUE TITLE
Support for Giscus?

### ISSUE THREAD

**Issue by thomaslevesque** [amzrk2/hugo-theme-fuji#105]
I'm currently considering using this theme, which looks really nice. Great job!

I see it supports [Utterances](https://utteranc.es/) natively, but not [Giscus](https://giscus.app/). Giscus is very similar to Utterances, but more recent, and uses Github discussions instead of issues.

I can easily customize the theme to use Giscus, but it would be great if it was supported natively!

**Comment by Bas-Man** [comment]
Perhaps you could contribute the feature in a PR.

**Comment by thomaslevesque** [comment]
Yeah. In fact, this might be the only way to do it right. I'm not sure the customization system will be enough to handle the theme switches, since I would need to change the Javascript code, not just the layout files.
